### PR TITLE
[CI:DOCS] Add play kube support docs

### PR DESCRIPTION
**Pull request by umohnani8**
Add a doc to outline which kube yaml fields the play
kube command currently supports.
This will be updated as more fields are supported in
the future.

Signed-off-by: Urvashi Mohnani <umohnani@redhat.com>

#### Does this PR introduce a user-facing change?

```release-note
Add doc outlining the kube yaml fields supported by podman play kube

```

### CODE REVIEW

**Comment by umohnani8** [comment]
@rhatdan @vrothberg PTAL

Will add fields for configmap and deployment kind as well.
I have only added fields for `spec` and not `metadata`, should we add those too?

File: docs/play_kube_support.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
+# Podman Play Kube Support
+
+This document outlines the kube yaml fields that we currently support with the **podman play kube** command.
```
Comment (vrothberg) [inline docs/play_kube_support.md:3]:
```suggestion
This document outlines the kube yaml fields that are currently support by the **podman play kube** command.
```

File: docs/play_kube_support.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+Note: **N/A** means that we will not support that option because it doesn't make sense in the podman world.
```
Comment (vrothberg) [inline docs/play_kube_support.md:5]:
```suggestion
Note: **N/A** means that the option cannot be supported in a single-node Podman environment.
```

**Comment by vrothberg** [comment]
> @rhatdan @vrothberg PTAL
> 
> Will add fields for configmap and deployment kind as well. I have only added fields for `spec` and not `metadata`, should we add those too?

SGTM

File: docs/play_kube_support.md (hunk 1 above)
Comment (TomSweeneyRedHat) [reply docs/play_kube_support.md:3]: Nice eye Valentin, no pronouns in man pages please!

**Comment by TomSweeneyRedHat** [comment]
LGTM once @vrothberg 's concerns are addressed.

**Comment by umohnani8** [comment]
@rhatdan this is ready

**Pull request merged**